service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsController.java

```diff
@@ -82,16 +90,19 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
         return respond(caseDetails);
     }
 
-    @PostMapping("/validate-pba-number/mid-event")
+    @PostMapping("/validate/mid-event")
```

Comment:
As mentioned in the defs file, how this going to work on prod? Is the code below going to run on the old data?

Reply:
I am not 100% sure here, please help me to understand this better. Users can't edit UploadedC2 at the moment, so there is no scenario for validating old data, is there?